src/components/datagrid/data_grid.tsx

```diff
@@ -208,31 +231,36 @@ export const EuiDataGrid: FunctionComponent<EuiDataGridProps> = props => {
   return (
     // Unsure why this element causes errors as focus follows spec
     // eslint-disable-next-line jsx-a11y/interactive-supports-focus
```

Comment:
this disable comment needs to stay directly above the `div[role="grid"]`: `{/* eslint-disable-next-line jsx-a11y/interactive-supports-focus */}`